kwc-tracking.html

```diff
@@ -436,14 +461,19 @@
             };
             /** The Polymer 1 behavior */
             Behavior = {
-                listeners: {
-                    'error-event': '_trackUserError',
-                    'initialize-tracking': '_initializeTracking',
-                    'page-view': '_trackPageView',
-                    'token-changed': '_tokenChanged',
-                    'tracking-event': '_trackBasicEvent'
+                attached () {
+                    window.addEventListener('tracking-event', _trackBasicEvent.bind(this));
```

Comment:
I think you will need to do ```this._trackBasicEvent = this._trackBasicEvent.bind.this; ``` before adding the listener, otherwise it doesn't get properly removed later.

Reply:
Ah thanks, good point. I've updated to make sure these are properly removed.